scripts/sysman/PROG.rst

```diff
@@ -792,7 +795,8 @@ domain on a device:
                allocate_memory(numPowerDomains * sizeof(${s}_pwr_handle_t))
            if (${s}DeviceEnumPowerDomains(hSysmanDevice, &numPowerDomains, phPower) == ${X}_RESULT_SUCCESS)
                for (pwrIndex = 0 .. numPowerDomains-1)
-                   ${s}_power_properties_t props
+                   ${s}_power_properties_t props {};
+                   props.stype = ${S}_STRUCTURE_TYPE_POWER_PROPERTIES;
```

Comment:
missing setting pNext to nullptr

Reply:
@matcabral : initializer `{}` will initialize all members to its default values, including pointers to nullptr.